src/test/java/uk/gov/hmcts/sptribs/exception/DocumentUploadOrDeleteExceptionTest.java

```diff
@@ -23,10 +23,12 @@
 import static uk.gov.hmcts.sptribs.testutil.TestConstants.DOCUMENT_UPLOAD_FAILURE_MSG;
 import static uk.gov.hmcts.sptribs.testutil.TestConstants.TEST_URL;
 
+
```

Comment:
In general, all the cleanup like removing unwanted lines and formatting and removing warnings can be done for all the files.